solana/bridge-escrow/programs/bridge-escrow/src/lib.rs

```diff
@@ -98,16 +92,23 @@ pub mod bridge_escrow {
             },
         );
 
-        let memo = "{\"forward\":{\"receiver\":\"0x4c22af5da4a849a8f39be00eb1b44676ac5c9060\",\"port\":\"transfer\",\"channel\":\"channel-52\",\"timeout\":600000000000000,\"next\":{\"memo\":\"my-custom-msg\"}}}".to_string();
+        let memo = "{\"forward\":{\"receiver\":\"\
+                    0x4c22af5da4a849a8f39be00eb1b44676ac5c9060\",\"port\":\"\
+                    transfer\",\"channel\":\"channel-52\",\"timeout\":\
+                    600000000000000,\"next\":{\"memo\":\"my-custom-msg\"}}}"
+            .to_string();
         let memo = memo.replace("my-custom-msg", &my_custom_memo);
```

Comment:
Why use replace? Use format instead. And also, how about formatting it better:

        let memo = format!(
            "{\"forward\":{\
                "receiver\":\"0x4c22af5da4a849a8f39be00eb1b44676ac5c9060\",\
                \"port\":\"transfer\",\
                \"channel\":\"channel-52\",
                \"timeout\":600000000000000,\
                \"next\":{\"memo\":\"{},{},{},{}\"}\
            }}",
            source_account.key(),
            destination_account.key(),
            authority.key(),
            token_program.key(),
        );

Add `[rustfmt::skip]` if fmt breaks the formatting.